src/Symfony/Component/Messenger/Event/WorkerRunningEvent.php

```diff
@@ -14,11 +14,11 @@
 use Symfony\Component\Messenger\Worker;
 
 /**
- * Dispatched after the worker processed a message or didn't receive a message at all.
+ * Events that are dispatched after the worker processed a message or didn't receive a message at all.
  *
  * @author Tobias Schultze <http://tobion.de>
  */
-final class WorkerRunningEvent
+abstract class WorkerRunningEvent
```

Comment:
This is a BC break. You can remove `final`, but you can't add `abstract`

Reply:
Thanks. Would dispatching only the child classes (WorkerBusy, WorkerIdle) as events instead of this one also be a BC break? Since existing listeners for this event would have to be adjusted? In that case I might need to add a command option to use the new events and emit a deprecation notice if it isn't passed.